packages/web-react/src/components/TextFieldBase/README.md

```diff
@@ -22,5 +22,5 @@ This is React implementation of the abstract component [TextFieldBase] for the p
 | `name`              | string                                                        | Input name                                                              |
 | `placeholder`       | string                                                        | Input placeholder                                                       |
 | `type`              | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | Input type                                                              |
-| `validationState`   | `success`, `warning`, `error`                                 | Type of validation state                                                |
+| `validationState`   | `success`, `warning`, `danger`, `error`                       | Type of validation state                                                |
```

Comment:
please link to docs/dictionaries and deprecation info

Reply:
done